source/api/commands/then.md

```diff
@@ -68,18 +70,45 @@ cy.get('button').then(($btn) => {
 })
 ```
 
+### The number is yielded
+
+```js
+cy.wrap(1).then((num) => {
+  cy.wrap(num)).should('equal', 1) // true
```

Comment:
syntax error `cy.wrap(num))` and this case really is also "subject is changed with another command", right